extension/src/test/unit/gradleDaemons.test.ts

```diff
@@ -273,6 +273,7 @@ describe(getSuiteName("Gradle daemons"), () => {
                 .catch(reject);
             // This call will return the correct results (longReply)
             gradleDaemonsTreeDataProvider.refresh();
+            await sleep(1000);
```

Comment:
what's changed upstream, causing you have to wait 1000ms?

Reply:
not sure what's changed there, some findings:

- See the current implementation: https://github.com/microsoft/vscode-gradle/blob/d2bb90190452b28ad41bc1b76895c3ca383cd268/extension/src/test/unit/gradleDaemons.test.ts#L267-L282
- this test case aims to test if a "urgent" refresh will skip the running process that fetches the daemon status (called "future result", `IDLE` here) and return the "current result" (`BUSY` here). 
- This case includes two parts: 
  - part 1: The `refresh()` call in line 275 will skip the `getChildren()` call in line 269 to return the "current result", 
  - part 2: The `getChildren()` call in line 277 can finish with "future result" successfully since it will not be interrupted by any `refresh()` call.

- when using the original test code and tested in VS Code 1.74.3, the debugger view shows the result is **not expected**, but the test case **passed**. Not sure why.
![image](https://user-images.githubusercontent.com/45906942/222381345-520fe18b-2fca-4b50-ba04-6f09f4060444.png)
![image](https://user-images.githubusercontent.com/45906942/222381449-1aa042fe-991e-4db1-b327-c43d92e84333.png)

- So I just sleep here to distinguish two parts, workaround here for now and make this test behavior more "manually".